src/anomalib/utils/visualization/__init__.py

```diff
@@ -4,11 +4,13 @@
 # SPDX-License-Identifier: Apache-2.0
 
 from .base import BaseVisualizer, GeneratorResult, VisualizationStep
+from .explanation import ExplanationVisualizer
```

Comment:
This is still to be discussed right?

Reply:
if we first had `ImageVisualizer`, would this one be something like `ImageLanguageVisualizer`, `MultimodalVisualizer`, `VlmVisualizer`, or something similar. Is it just text explanation?